Added new option - Only show button in Space Center

Gribbleshnibit8 opened the pull request:
Because the AppLauncher can get cluttered and this is a mod that I usually change once and never touch again.

MOARdV commented on Source-Code/SettingsGui.cs:
```diff
@@ -173,18 +181,26 @@ public void Awake()
+
+	            if (onlyInSpaceCenter && HighLogic.LoadedSceneIsFlight)
+	            {
+		            ApplicationLauncher.Instance.RemoveModApplication(appLauncherButton);
```
Have you tested repeatedly going from flight to space center with this code?  There seems to be some strange behavior with removing launcher buttons in KSP 1.0.  I think it might be better to hide the button during GameScenes.FLIGHT by changing appLauncherButton.VisibleInScenes when onlyInSpaceCenter changes, although I haven't tested that code.

Gribbleshnibit8 commented:
I tested it a couple times back and forth to make sure it showed up and went away when changing the option. Was not aware that there is a VisibleInScenes value to toggle, which would of course be better. This was my first foray into the wilds of messing with applauncher stuff in any way.

Gribbleshnibit8 commented:
OK, changed it to use the AppScenes field and that seems to work much better. Changed scenes multiple times between flight, space center, and tracking station just for testing, toggling option occasionally as I did. Nothing bad seemed to happen that I could tell.
